src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
`catalog` is a BEM block so fix it 
![image](https://user-images.githubusercontent.com/97547824/211077470-340420a0-bc73-4028-9888-d8d41ec74c36.png)